vscode/src/views/TestViewController.ts

```diff
@@ -35,7 +35,7 @@ export class NbTestAdapter {
     private started: boolean = false;
 
     constructor() {
-        this.testController = tests.createTestController('apacheNetBeansController', 'Apache NetBeans');
+        this.testController = tests.createTestController('apacheNetBeansController', 'Oracle Java VS code');
```

Comment:
1. The api doc for [createTestController](https://code.visualstudio.com/api/references/vscode-api#tests) mentions that the `id` (i.e. first arg) should be globally unique.
    - Perhaps we should use: `extConstants.ORACLE_VSCODE_EXTENSION_ID + '.testController'`
2. The user-visible label needs to be in line with our extension name , i.e. `Java`.